incubator/hnc/internal/validators/object_test.go

```diff
@@ -273,8 +277,9 @@ func TestUserChanges(t *testing.T) {
 			},
 		},
 	}, {
-		name: "Deny deletions of propagated objects",
-		fail: true,
+		name:       "Deny deletions of propagated objects",
+		fail:       true,
+		isDeleting: true,
```

Comment:
This is actually backwards - if the namespace containing the object is being deleted, we should _allow_ the propagated object to be deleted too (since we're not trying to block namespace deletion). BTW this implies that we should have two variants of this test - one with the NS being deleted, and one without.